### PR TITLE
Latest stack lts

### PR DESCRIPTION
**Pull request by andreyk0**
- stratosphere points to upstream master (our issues seem to be fixed there)

### CODE REVIEW

**Comment by andreyk0** [comment]
* amazonka - same
* stratosphere - latest upstream `master` which seems to include the fixes we needed
   * Notable other change is the removal of the giant union "resource properties" type

File: snapshots/er-lts-13.13.0.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
+resolver: lts-13.13
+name: er-lts.13.13.0.yaml
+
+packages:
+# Upstream 'master' addresses the issues we had in the PR
+# but it's not on stackage yet.
+# https://github.com/frontrowed/stratosphere/issues/111
```
Comment (fiadliel) [inline snapshots/er-lts-13.13.0.yaml:7]: Can't you refer to the version on hackage?

```
extra-deps:
- stratosphere-0.34.0
```

Comment (andreyk0) [reply snapshots/er-lts-13.13.0.yaml:7]: Does matter if this master commit SHA is exactly the same as that tag? Do you still prefer hackage version?

Comment (fiadliel) [reply snapshots/er-lts-13.13.0.yaml:7]: Not especially, though it is possible to get prebuilt packages using hackage version? You probably know what's usual, better than me.

Comment (andreyk0) [reply snapshots/er-lts-13.13.0.yaml:7]: AFAIK there are no pre-built **binary** packages with stack. It downloads source and compiles/caches locally. 
If I update this reference all that would practically change is the mechanics of how it gets source code (GH download vs tgz download from hackage).

Anyway, I guess there's some aesthetic value to this, so, I'll change it :)

**Comment by andreyk0** [comment]
Oh, wait, that didn't quite work ...

**Comment by andreyk0** [comment]
Ok, seems fine now, extra `-` threw me off :)

**Comment by andreyk0** [comment]
Ah, need to go off hackage rails anyway, we just re-opened https://github.com/freckle/stratosphere/issues/111 issue, I got excited too early :) Saw it marked as fixed but it's only partially fixed with the latest updates.

**Comment by andreyk0** [comment]
Actually, I think there's a different way to hack around that now.
I'll merge this and will explore in a different PR in the CF stacks repo.

**Pull request merged**